cpu/esp_common/esp-now/esp_now_netdev.c

```diff
@@ -633,7 +633,7 @@ static int _send(netdev_t *netdev, const iolist_t *iolist)
     uint8_t* _esp_now_dst = NULL;
 
     for (uint8_t i = 0; i < ESP_NOW_ADDR_LEN; i++) {
-        if (((uint8_t*)iolist->iol_base)[i] != 0xff) {
+        if (iolist->iol_base != NULL && ((uint8_t*)iolist->iol_base)[i] != 0xff) {
```

Comment:
parenthesis missing around expressions.

Reply:
Ok, even though the compiler seems to be happy and doesn't warn.

Reply:
Changed it.

Reply:
> Ok, even though the compiler seems to be happy and doesn't warn.

It's not about the compiler but about [readability / style](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#indentation-and-braces) ;-).